src/pages/starter-kit/reference-app.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Reference applications
```

Comment:
I think we should title that like Microsoft F&O Accelerator - Reference application? or maybe Reference Application - Microsoft F&O Accelerator ?